examples/fpga/artya7-100/README.md

```diff
@@ -0,0 +1,46 @@
+# Ibex RISC-V Core SoC Example
+
+This is a simple example of how Ibex can be utilized.
+The core is not connected to a bus, but to the SRAM directly, which is initialized
+by the `led.vmem` file from the software section.
+
+## Requirements
+
+### Tools
+
+  - RISC-V compiler
+  - srecord
+  - `fusesoc` and its dependencies
+  - Xilinx Vivado
+
+## Build
+
+### Software
+
+First the software must be built. Go into `sw/led` and call:
+
+```
+make CC=/path/to/RISC-V-compiler
```

Comment:
Can you make this `/path/to/riscv32-unknown-elf-gcc` to make it easier to find the right compiler for people?

Reply:
That can also cause confusion unfortunately - multilib builds will use riscv64-unknown-elf-gcc but may include libs for ilp32.

Reply:
So `/path/to/riscvXX-unknown-elf-gcc`?

Reply:
Well, even the *unknown* part is not really standard (unfortunately), with the rather commonly used [Eclipse prebuilts](https://gnu-mcu-eclipse.github.io/toolchain/riscv/install/) defaulting to *none* here. I'd go for either riscv32-unknown-elf-gcc (which is what you get with a SiFive toolchain source build) or just leave it as you had it.

Reply:
I just checked the Eclipse site, looks like they now use `riscv-none-embed-gcc`.
I will just put in some examples of what this path could look like. I would assume that the chances are high that if someone finds the way to Ibex Arty example, they are able to set the path to the right compiler correctly.